doc/guides/CustomStartupGuide.md

```diff
@@ -452,8 +455,85 @@ Follow these instructions to update devices.
    ```bash
    [{"apiVersion":"v2","statusCode":201,"id":"fb5fb7f2-768b-4298-a916-d4779523c6b5"}]
    ```
+</details>
+
+<details>
+<summary><strong>Auto-Discovery</strong></summary>  
+
+<br/>
+
+ONVIF devices support WS-Discovery, which is a mechanism that supports probing a network to find ONVIF capable devices.  Refer to [Auto Discovery](../auto-discovery.md) for detailed information on the auto-discovery mechanism.  
+>**NOTE:** Ensure that the cameras are all installed and configured before attempting discovery.    
+
+> For `Netscan`, there is a one line command to determine the `DiscoverySubnets` of your current machine:
+> ```shell
+> ip -4 -o route list scope link | sed -En "s/ dev ($(find /sys/class/net -mindepth 1 -maxdepth 2 -not -lname '*devices/virtual*' -execdir grep -q 'up' "{}/operstate" \; -printf '%f\n' | paste -sd\| -)).+//p" | grep -v "169.254.0.0/16" | sort -u | paste -sd, -
+> ```
+> Example Output: `192.168.1.0/24`
 
- Map credentials using the `map-credentials.sh` script.  
+>**NOTE:** Alternatively, for `netscan` you can set the `DiscoverySubnets` automatically _after_ the service has been deployed by running the [bin/configure-subnets.sh](../utility-scripts.md#configure-subnetssh) script
+
+#### 1. Discovery Configuration
```

Comment:
I think that many of the parts of this section need to go before startup, otherwise the changes won't take effect.